### PR TITLE
#3 Implement a discount property in Product

### PR DESCRIPTION
**Pull request by catalinsuceveanu**
Please review my implementation

### CODE REVIEW

File: src/main/java/ro/siit/Product.java [hunk 1]
```diff
@@ -7,10 +7,16 @@
 
     private String name;
     private BigDecimal price;
+    private double discount;
```
Comment (danutchindris) [inline src/main/java/ro/siit/Product.java]: Maybe it would have been better to use `BigDecimal` here.

**Review by danutchindris — CHANGES_REQUESTED** [review]
Please see my review comments.

File: src/main/java/ro/siit/Product.java (hunk 1 above)
Comment (catalinsuceveanu) [reply src/main/java/ro/siit/Product.java]: exagerati sefu'

**Review by danutchindris — APPROVED** [review]
Great implementation.
Please merge the branch to main.

**Pull request merged**